### PR TITLE
More mutation toxins for green extracts

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

title

Dunno if it needs approval but I got it anyway

## Why It's Good For The Game

Moffamphetamine
Totally not stolen joke
## Changelog
:cl:
add: A whole bunch of new mutation toxin reactions for green slime extracts
/:cl:

### CODE REVIEW

**Comment by doguinh0** [comment]
please do it

**Comment by Nikitaw99** [comment]
Yes please
im all for moth and felinid mutation toxin

**Review by Username0103 — APPROVED** [review]

**Comment by itseasytosee** [comment]
I have quite a few complaints about this.
1. shoot someone with plasma man mutation toxin for a quick burn and kill.
2. Felinids are just humans with ears graphed on them and can already be made in-game.
3. There are already ways to become a golem/pod person/fly that are much easier to do. (golem can be done by using a luminescent feature.
4. Don't let people become abductors are you even thinking about the implications of that.

**Comment by TiviPlus** [comment]
> I have quite a few complaints about this.
> 
> 1. shoot someone with plasma man mutation toxin for a quick burn and kill.
> 2. Felinids are just humans with ears graphed on them and can already be made in-game.
> 3. There are already ways to become a golem/pod person/fly that are much easier to do. (golem can be done by using a luminescent feature.
> 4. Don't let people become abductors are you even thinking about the implications of that.

1. Lings can already do that shiftstart instantly, while mutation toxin takes time to act and gives you unique warnings during this time
2. Offtopic, dont even start that shit here
3. OK so what
4.??? It just makes them into a ayylmao without the antag I dont see the issue

**Comment by itseasytosee** [comment]
1 Lings actually can not transform people into a plasma man they get the message
image

2 and 3 these are redundant features that add nothing to the game.

4 abductors have a chat separate from the rest of the crew for a reason. being able to turn into an abductor is as easy as having green slimes and uranium. It topples it the fuck over.

**Comment by ArcaneDefence** [comment]
Charged black can already turn you into an abductor, but it's one among many possible results.

**Comment by itseasytosee** [comment]
with this, why would you ever use a charged black now anyways?

**Comment by ExcessiveUseOfCobblestone** [comment]
@doguinh0 please do not use the review feature unless you're reviewing code.

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -58,6 +58,48 @@
...
 	required_container = /obj/item/slime_extract/green
 
+/datum/chemical_reaction/slime/slimemoth
+	results = list(/datum/reagent/mutationtoxin/moth = 1)
+	required_reagents = list(/datum/reagent/ash = 1)
+	required_other = TRUE
+	required_container = /obj/item/slime_extract/green
+
+/datum/chemical_reaction/slime/slimeplasma
+	results = list(/datum/reagent/mutationtoxin/plasma = 1)
```
Comment (Fikou) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:68]: OH NO NO NO

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 2]
```diff
@@ -58,6 +58,48 @@
...
+	required_container = /obj/item/slime_extract/green
+
+/datum/chemical_reaction/slime/slimefelinid
+	results = list(/datum/reagent/mutationtoxin/felinid = 1)
+	required_reagents = list(/datum/reagent/consumable/cream = 1)
+	required_other = TRUE
+	required_container = /obj/item/slime_extract/green
+
+/datum/chemical_reaction/slime/slimepod
+	results = list(/datum/reagent/mutationtoxin/fly = 1)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:86]: copypasted code detected

**Comment by Shadowflame909** [comment]
I've seen this PR 3 times before. And I'm pretty sure these mutation toxins existed in the first place. Anyone know why they were removed?

Another thing is, I don't recommend this change. Plasma-meme toxin and all the other ones are strange seed only for a reason. They'll cause a lot of drama and salt in the normal rounds. Like literally, rogue xenobiologist starts turning people into cat-girls. We're getting suicide HoP-Line.

**Comment by Spepbuthewranglesgithub** [comment]
plasmaman/felinid mutation toxins are just terrible ideas, the former means death, the latter means more death and a veritable singularities worth of salt.


turning into an ayy is pretty cool though.

**Comment by RaveRadbury** [comment]
> I've seen this PR 3 times before. And I'm pretty sure these mutation toxins existed in the first place. Anyone know why they were removed?

**History Time**
Stable mutation toxins #20677 - Author: XDTM , _merged_ Oct 2, 2016
Xenobiology Species Rework #34166 - Author: XDTM, _merged_  Jan 15, 2018
Adds mutation toxin recipes for roundstart race. #39759 - Author: Wilchenx, _closed_ Aug 19, 2018

**Comment by Shadowflame909** [comment]
Rave with the save! Mutation Toxins sure do get changed a lot, huh.

**Comment by Memedoktor** [comment]
Oh man this is pretty cool, getting more mutation toxins can be a fun way to just shift yourself around. Also can't you already order plasmeme suits?

**Comment by RaveRadbury** [comment]
> Oh man this is pretty cool, getting more mutation toxins can be a fun way to just shift yourself around. Also can't you already order plasmeme suits?

Originally this was balanced by making it an _advanced_ mutation toxin. Unexpected species change to plasmaman is pretty much a death sentence and it needs to be balanced accordingly.

**Comment by itseasytosee** [comment]
@Memedoktor the idea is that you can forcibly inject it into people for an easy kill.

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm (hunk 2 above)
Comment (TiviPlus) [reply code/modules/reagents/chemistry/recipes/slime_extracts.dm:86]: its like 2am ok

**Comment by TiviPlus** [comment]
The only real issue I'm getting here is that the plasmeme toxin is too OP, I'll look into nerfing it somehow

**Comment by TiviPlus** [comment]
> 2 and 3 these are redundant features that add nothing to the game.

This PR isnt about Felinids, if you want to complain take it to the NTR hut and stop dragging this shit off topic
 
> 4 abductors have a chat separate from the rest of the crew for a reason. being able to turn into an abductor is as easy as having green slimes and uranium. It topples it the fuck over.

???????
You can become an ayylmao damn near roundstart if you find a ruin, never has been an issue
You can convert the entire crew into ayylmaos as ling (Tried and tested) never has been an issue
Ayylmaos are mute

**Comment by optimumtact** [comment]
I have denied these before, please dont' add things to force change peoples species. I don't think it's worth the inevitable complaints.

**Pull request closed without merging**